web/src/components/Prelude.res

```diff
@@ -71,6 +71,14 @@ module URLSearchParams = {
   let current = () => windowLocationSearch->make
 }
 
+let defaultUpdateFilters = (filters: string) => {
+  let current = URLSearchParams.current()->URLSearchParams.toString
+  switch current == filters {
+  | true => RescriptReactRouter.push("/changes/openstack?" ++ filters)
```

Comment:
If I recall correctly, ReactRouter.push does not tricker the url listener of the query builder.